### DISCUSSION TITLE
how do i make games available on the EGL as well?

### DISCUSSION THREAD

**Discussion by user4302** [legendary-gl/legendary#219] [Q&A] [answered]
I have a game installed and working on legendary, 
but it is said to not be installed when i check EGL.

I ran 'legendary -y egl-sync' and the output was
![image](https://user-images.githubusercontent.com/32593782/105799328-46b8bd00-5fba-11eb-969e-43d764d90026.png)

**Comment by derrod** [comment, marked as answer]
What game? Also make sure to restart EGL completely (it runs in the background by default). If it still doesn't show up check the EGL Manifests folder (`/ProgramData/Epic/EpicGamesLauncher/Data/Manifests` by default) and see if any of the files belong to the game you installed via legendary.

**Reply by user4302** [reply]
Hyper Light Drifter [Parakeet]

i just ran 'legendary list-installed --json'
and found "egl_guid": "6269465195AD407EA6E069B1AC1162BD"
and there is  a file in "/ProgramData/Epic/EpicGamesLauncher/Data/Manifests" that matches that guid.

i rebooted the pc and opened EGL and the game was still not listed as installed.

btw i "installed" the game by copying the folder to my pc and using "legendary import-game"
the game works properly to my knowledge when i use "legendary launch Parakeet"

"/ProgramData/Epic/EpicGamesLauncher/Data/Manifests" also contains a folder called "pending"
and the GUID for this game is not inside this folder, but it does have 2 other games that i have installed as well as  a bunch of other GUIDs that im not familiar with

**Reply by derrod** [reply]
Check that the path in the manifest file is cocrect, otherwise the Epic Launcher won't list the game. If it still doesn't then I don't really know, the export has been done but it isn't picking up the file for some reason.

**Reply by user4302** [reply]
i do not have the ".egstore" folder in the game folder, my best assumption is that that is the issue. should i try to manually create the folder and necessary files?

**Reply by derrod** [reply]
That would explain why the game is not listed, but not why that folder doesn't exist... when `egl_export()` is called it both creates the folder and files.

**Reply by user4302** [reply]
~im guessing that "egl_export()" is function that is called when "legendary -y egl-sync" is run?~
so the ".egstore" folder should be created when i run "legendary -y egl-sync" ?

**Reply by derrod** [reply]
The `.egstore` folder should be created when the game is synced to the egs for the first time. If you enabled syncing before importing then it should have been created when importing.

**Reply by user4302** [reply]
thank you. this worked. 
uninstalled using legendary.
enabled sync 
imported using legendary.
opened EGL, and it was listed as installed

**Comment by user4302** [comment]
correct me if im wrong, the "syncing" you mentioned is this right? [EGL Sync enabled: True]
![image](https://user-images.githubusercontent.com/32593782/105813221-8b058680-5fd5-11eb-9b11-1f4d76333371.png)
will it ever get automatically disabled?
will the games automatically be "added" to EGL from now on?
